### DISCUSSION TITLE
Pywinauto Function That Traces a Line

### DISCUSSION THREAD

**Discussion by TRSlade** [pywinauto/pywinauto#1271] [Q&A]
Is there a function in Pywinauto that will automatically move my cursor along a line? I have a DWG floor plan that I would like Python to outline room by room. Thanks for the help.

**Comment by vasily-v-ryabov** [comment]
There is method `.draw_outline()`. Maybe it's not sufficient for part of the element. The rest can be done with mouse methods like `.move_mouse_input()`, `.press_mouse_input()` and `.release_mouse_input()`.
